django_netjsonconfig/base/admin.py

```diff
@@ -11,6 +12,7 @@
 from django.template.response import TemplateResponse
 from django.urls import reverse
 from django.utils.translation import ugettext_lazy as _
+from django_netjsonconfig.models import Config
```

Comment:
I just realized you cannot do this. You would break extensibility because you are importing a concrete model into base classes. See the next comment on how to fix this, remove this import please.